leakcanary-android-core/src/main/java/leakcanary/internal/activity/db/Cursors.kt

```diff
@@ -20,7 +20,7 @@ internal inline fun <R> Cursor.use(block: (Cursor) -> R): R {
       null -> close()
       else -> try {
         close()
-      } catch (ignoredCloseException: Throwable) {
+      } catch (_: Throwable) {
```

Comment:
interesting, not sure I buy that `_` works betters than a variable name that explicitly says "ignored", what's your reasoning? Just applying automatic kotlin fixes?

Reply:
No use of this `ignoredCloseException`, can be ignored with `_`, I see the auto suggestion by AS.

Reply:
yeah, that's the point, it's called "ignored" so signal that it's going to be ignored, ie not used.